data/networks.ts

```diff
@@ -28,12 +32,33 @@ export const l1Networks = {
 export type L1Network = (typeof l1Networks)[keyof typeof l1Networks];
 
 export type EraNetwork = L2Network & {
-  id: 324 | 280 | 270;
+  id: number;
   rpcUrl: string;
-  blockExplorerApi: string;
+  blockExplorerApi?: string;
   faucetUrl?: string;
+  getTokens: () => Token[] | Promise<Token[]>;
 };
 export const eraNetworks: EraNetwork[] = [
+  // See official documentation for in-memory node setup details: https://era.zksync.io/docs/tools/testing/
+  // Also see the guide in the README.md file in the root of the repository.
+  {
+    id: 260,
+    key: "era-local",
+    name: "zkSync Era Local",
+    shortName: "Era Local",
+    rpcUrl: "http://localhost:8011",
```

Comment:
hmmm..... in memory node runs on a different port than local-setup. confusing indeed. Maybe we should add both to make it clear?

Reply:
caveat - in memory node do not allow bridging, given it doesn't run the L1

Reply:
Portal already supports running without L1 network